### PR TITLE
feat(general-equipment-editor): add remove button

### PR DESCRIPTION
**Pull request by marcvanraalte**

### CODE REVIEW

File: src/editors/substation/general-equipment-editor.ts [hunk 1]
```diff
@@ -94,11 +116,21 @@ export class GeneralEquipmentEditor extends LitElement {
           @click=${() => this.openEditWizard()}
         ></mwc-icon-button>
       </abbr>
+      <abbr slot="action" title="${translate('remove')}">
+        <mwc-icon-button
```
Comment (JakobVogelsang) [inline src/editors/substation/general-equipment-editor.ts:120]: Using `action-icon` `mwc-icon-button` looks strange. That is why all other editor use `mwc-fab` instead.
Same is true for the edit button. Sorry that slipped through the review

**Review by JakobVogelsang — CHANGES_REQUESTED** [review]

File: src/editors/substation/general-equipment-editor.ts (hunk 1 above)
Comment (pascalwilbrink) [reply src/editors/substation/general-equipment-editor.ts:120]: It looks like they're being mixed. sometimes mwc-icon-button, sometimes mwc-fab.
If you look at the function-editor (for example), that one is using mwc-icon button.
the powertransformer-editor (for example) is using the mwc-fab.

@JakobVogelsang, we can change it to mwc-fab for now, but maybe it's better to change it everywhere?

Comment (pascalwilbrink) [reply src/editors/substation/general-equipment-editor.ts:120]: We found out what you meant with action-icon > mwc-icon-button :) changed it in the code. should look good now

Comment (JakobVogelsang) [reply src/editors/substation/general-equipment-editor.ts]: Nice. Thx

**Review by JakobVogelsang — APPROVED** [review]
LGTM

**Pull request merged**